nvim/after/plugin/lualine.lua

```diff
@@ -1,3 +1,7 @@
+if vim.g.vscode then
+  return
+end
```

Comment:
This shows up a lot - I sometimes use VSCode with the neovim plugin and this prevents my settings from messing with it.